app/views/issues/_form.html.erb

```diff
@@ -29,6 +29,10 @@
     <%= hidden_field :issue, :original_updated_at, value: @issue.updated_at.to_i %>
   <% end %>
 
+  <% if local_assigns.has_key?(:qa) && qa %>
+    <%= hidden_field nil, :return_to, value: :qa %>
+  <% end %>
+
```

Comment:
Isn't this flag something a) the controller should set; or b) we can infer from the local `controller` variable?

I don't think we need a local "view" variable that only is present some times... feels like overdesigning